Source/WebCore/dom/Document.h

```diff
@@ -2219,6 +2221,8 @@ class Document
 
     ListHashSet<Ref<Element>> m_topLayerElements;
 
+    Vector<RefPtr<HTMLElement>> m_popoverStack;
```

Comment:
I don't think the pointers can be null so we probably want to use `Ref<>`, not `RefPtr<>`.

Reply:
Done.